samples/snippets/speech_model_adaptation_beta.py

```diff
@@ -0,0 +1,97 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START speech_transcribe_with_model_adaptation]
+import google.auth
+from google.cloud import speech_v1p1beta1 as speech
+
+
+def transcribe_with_model_adaptation(project_id, location, storage_uri):
+
+    """
+    Create`PhraseSet` and `CustomClasses` to create custom lists of similar
+    items that are likely to occur in your input data.
+    """
+
+    # Create the adaptation client
+    adaptation_client = speech.AdaptationClient()
+
+    # The parent resource where the custom class and phrase set will be created.
+    parent = f"projects/{project_id}/locations/{location}"
+
+    # Create the custom class
+    custom_class_response = adaptation_client.create_custom_class(
+        {
+            "parent": parent,
+            "custom_class_id": "seattlerestaurants",
+            "custom_class": {
+                "items": [
+                    {"value": "sushido"},
+                    {"value": "altura"},
+                    {"value": "taneda"},
+                ]
+            },
+        }
+    )
+
+    # Create the phrase set
+    phrase_set_response = adaptation_client.create_phrase_set(
+        {
+            "parent": parent,
+            "phrase_set_id": "restaurantphrasesets",
+            "phrase_set": {
+                "boost": 10,
+                "phrases": [{"value": "Visit restaurants like ${seattlerestaurants}"}],
+            },
+        }
+    )
+
+    # Next section shows how to use the newly created custom
+    # class and phrase set to send a transcription request with speech adaptation
+
+    # Speech adaptation configuration
+    speech_adaptation = speech.SpeechAdaptation(
+        phrase_sets=[phrase_set_response], custom_classes=[custom_class_response]
+    )
+
+    # speech configuration object
+    config = speech.RecognitionConfig(
+        encoding=speech.RecognitionConfig.AudioEncoding.LINEAR16,
+        sample_rate_hertz=24000,
+        language_code="en-US",
+        adaptation=speech_adaptation,
+    )
+
+    # The name of the audio file to transcribe
+    # storage_uri URI for audio file in Cloud Storage, e.g. gs://[BUCKET]/[FILE]
+
+    audio = speech.RecognitionAudio(uri=storage_uri)
+
+    # Create the speech client
+    speech_client = speech.SpeechClient()
+
+    response = speech_client.recognize(config=config, audio=audio)
+
+    for result in response.results:
+        print("Transcript: {}".format(result.alternatives[0].transcript))
+
+
+# [END speech_transcribe_with_model_adaptation]
```

Comment:
After the region tag please `return` the transcript so it can be checked in the test.